### DISCUSSION TITLE
I can't filter timestamp

### DISCUSSION THREAD

**Discussion by lazpuzzle** [apache/superset#19477] [Q&A / Help]
### Discussed in https://github.com/apache/superset/discussions/19427

<div type='discussions-op-text'>

<sup>Originally posted by **lazpuzzle** March 30, 2022</sup>
![image](https://user-images.githubusercontent.com/40905829/160799537-310e93bc-5472-4968-821d-8f9369408022.png)

how i can config this timestamp. the data import from opensearch</div>

**Comment by srinify** [comment]
Hey there, moving to Discussions as this isn't a bug per say :)

Some advice:

- Either modify the format in the underlying database itself
- Or use a calculated column to transform specific columns: <img width="1648" alt="Screen Shot 2022-03-31 at 5 05 38 PM" src="https://user-images.githubusercontent.com/801507/161149409-08bb0aec-e4dc-4e25-9d40-caf054801a4d.png">

**Reply by lazpuzzle** [reply]
sorry again it doesn't clear for me.
my data connect from opensearch 
the timestamp columns has value is
in this result i query by opendistro sql
![image](https://user-images.githubusercontent.com/40905829/161179211-c50b977d-6c58-4fd6-a07e-7acd1f32b2a8.png)

and then connect to superset timestamp auto convert to
![image](https://user-images.githubusercontent.com/40905829/161179277-8b1b32fb-3dfb-4073-a632-6489c775f992.png)


when i filter in chart and see query sql it filter by
![image](https://user-images.githubusercontent.com/40905829/161179372-d07830b3-2623-4d89-9d29-67634f3f9b8d.png)


how i manage it to the same pattern
